src/plugins/intel_cpu/thirdparty/ACLConfig.cmake

```diff
@@ -119,6 +119,8 @@ elseif(NOT TARGET arm_compute::arm_compute)
 
     set(extra_cxx_flags "${CMAKE_CXX_FLAGS} -Wno-undef")
     if(MSVC64)
+        # Recommended Win ARM arch build in https://arm-software.github.io/ComputeLibrary/latest/how_to_build.xhtml
```

Comment:
I don't see such recommendations.
For Windows build arm64v8a is supposed as example, IMO

Do experience issues when you build for arm64v8.2a ?

Reply:
@ilya-lavrenov yes, this was default arch, but for fp16 case it's not work because we can't build kernel  
`arm_conv::winograd::weight_transform::a64_fp16_4x4_3x3` and other same kernels in ComputeLibrary (alas FP16 temproray disable, but I opened issue to ACL:  https://github.com/ARM-software/ComputeLibrary/issues/1135)
What I need to do in this case?

Reply:
ok, as I see they confirmed armv8a is supported only. Thank you for the ticket.